### ISSUE TITLE
Fails to start on Ubuntu 24

### ISSUE THREAD

**Issue by iparnamaa** [neutralinojs/neutralinojs#1258]
Neutralino _x64 binary fails to start on Ubuntu 24.04.
Apparently it searches for libwebkitgtk-4.0, but Ubuntu 24.04 installs libwebkitgtk-4.1

**Specifications**  
 - OS: Ubuntu 24.04
 - Neutralinojs version: [e.g. v5.0.0]

**Comment by raluvy95** [comment]
Same issue here. Few Ubuntu-based distros such as Linux Mint have upgraded to use Ubuntu 24 repo

**Comment by rhythnic** [comment]
You can install libwebkitgtk-4.0.  I'm running NeutralinoJjs on Ubuntu 24.

edit /etc/apt/sources.list
```bash
sudo nano /etc/apt/sources.list
```
add a new line in the document and paste this into the new line:

```
deb http://gb.archive.ubuntu.com/ubuntu jammy main
```

```bash
sudo apt update
sudo apt install libwebkit2gtk-4.0-dev
```

After installation, undo the change to /etc/apt/sources.list.  Leaving the line in there can cause issues with updates.

References:
- https://stackoverflow.com/questions/62301866/how-to-install-the-libwebkitgtk-package-on-ubuntu-20-04-lts
- https://github.com/bambulab/BambuStudio/issues/3973#issuecomment-2085476683

**Comment by shalithasuranga** [comment]
Fixed via #1307 (see the last comment for more info). Thanks so much for reporting this issue and starting a discussion :tada:
